_episodes/01-intro.md

```diff
@@ -220,4 +220,29 @@ weight in kilograms is now: 65.0
 > {: .solution}
 {: .challenge}
 
+> ## Seeing Data Types
+>
+> Python can show a variable's data type using type(variable). 
```

Comment:
Since we will move introduction of `type()` to the main part of the episode, would you please remove the line above.

Reply:
Though we (the maintainers) still need to open a PR to introduce `type()` earlier in the episode, we would like you to remove the line `> Python can show a variable's data type using type(variable).` , so when we have both PRs merged, `type()` won't be introduced a second time.